### DISCUSSION TITLE
How to use wails without npm?

### DISCUSSION THREAD

**Discussion by decentral1se** [wailsapp/wails#3418] [General]
I basically just need a server (auto-)reloading the `index.html` on the front-end. I think it has to do with replacing the `"frontend:dir": "..."` from the `wails.json` but is there something else to deal with? Any tips welcome!

Thanks for all the work, Wails is amazin' 🤘

**Comment by decentral1se** [comment]
Adapted from `wails init -t plain -n MyProject` 👍

**Comment by leaanthony** [comment]
Just edit the values with npm in `wails.json`. This may help you: https://wails.io/docs/guides/manual-builds/#build-process
